lib/survey_gizmo/configuration.rb

```diff
@@ -41,22 +41,28 @@ def reset!
   end
 
   class Configuration
-    DEFAULT_API_VERSION = 'v4'
+    DEFAULT_API = :surveygizmo
+    DEFAULT_API_VERSION = 'v4'.freeze
     DEFAULT_RESULTS_PER_PAGE = 50
     DEFAULT_TIMEOUT_SECONDS = 300
     DEFAULT_REGION = :us
-    DEFAULT_LOCALE = 'English'
+    DEFAULT_LOCALE = 'English'.freeze
+
+    APIS_INFO = {
+      surveygizmo: 'restapi.surveygizmo',
+      alchemer: 'api.alchemer'
+    }.freeze
 
     REGION_INFO = {
       us: {
-        url: 'https://restapi.surveygizmo.com',
+        tld: 'com',
```

Comment:
the only part of the url that is region-dependent is the tld so changed the key and the value is now just the tld instead of being the entire url